files/chefdk-env-customization/windows/chefdk_env_customization.rb

```diff
@@ -0,0 +1,52 @@
+## Environment hacks for running Ruby with ChefDK ##
+# ENV['HOME'] is not set by default on Windows. We need to set this to
+# something sensible since a lot of Ruby code depends on it. It is important 
+# for this directory to exist and be available, so we are introducing logic
+# here to pick a working HOME
+#
+# You can find this file in the repo at https://github.com/chef/omnibus-chef
+
+if !ENV['HOME'] || !File.exists?(ENV['HOME'])
+  old_home = ENV['HOME']
+  found = false
+  alternate_homes = []
+  alternate_homes << "#{ENV['HOMEDRIVE']}#{ENV['HOMEPATH']}" if ENV['HOMEDRIVE']
+  alternate_homes << "#{ENV['HOMESHARE']}" if ENV['HOMESHARE']
```

Comment:
as discussed in standup, lets not use homeshare, but if it is set, we should print a message saying like "I see you have a homeshare, that sucks man. you can explicitly use it if you want, but caching on a network share is going to be a poor experience."